core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -7342,6 +7350,10 @@ public static final class Name {
     public static final String FUSE_JNIFUSE_ENABLED = "alluxio.fuse.jnifuse.enabled";
     public static final String FUSE_SHARED_CACHING_READER_ENABLED
         = "alluxio.fuse.shared.caching.reader.enabled";
+    public static final String FUSE_JNIFUSE_LIBFUSE_VERSION =
```

Comment:
Originally the libfuse version property key was under section "Standalone FUSE process related properties", which is not correct.